packages/http-client-csharp/generator/Microsoft.Generator.CSharp.TestPlugin/src/Microsoft.Generator.CSharp.TestPlugin.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Generator.CSharp.ClientModel</PackageId>
```

Comment:
lets rename this project to just TestPlugin drop the mgc since its mimicing a 3p plugin we shouldn't name it mgc